cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -145,18 +145,23 @@ public KafkaBrokerConfigurationBuilder withCruiseControl(String clusterName, Cru
     /**
      * Configures the Strimzi Metrics Reporter. It is set only if user enables Strimzi Metrics Reporter.
      *
-     * @param model     Strimzi Metrics Reporter configuration
+     * @param model Strimzi Metrics Reporter configuration
      *
      * @return Returns the builder instance
      */
-    public KafkaBrokerConfigurationBuilder withStrimziMetricsReporter(MetricsModel model)   {
+    public KafkaBrokerConfigurationBuilder withStrimziMetricsReporter(MetricsModel model) {
+        printSectionHeader("Strimzi Metrics Reporter configuration");
```

Comment:
Maybe the formatting can be improved a bit ...

```suggestion
        printSectionHeader("Strimzi Metrics Reporter configuration");
        
```